src/components/Carousel/index.tsx

```diff
@@ -26,6 +26,7 @@ export default class Carousel extends React.Component<CarouselProps, CarouselSta
     static displayName = 'Carousel';
 
     static defaultProps: CarouselProps = {
+        ariaLabel: 'Carousel',
```

Comment:
This change will affect all existing consumers so it would be considered a breaking change. We need to avoid breaking changes unless there is a good reason not to. A simple way to mitigate it would be to only add the attribute if provided instead of providing a default value. Please make the change and I will approve it.